rental.rb

```diff
@@ -0,0 +1,11 @@
+class Rental
+  attr_accessor :date, :book, :person
+
+  def initialize(date)
+    @date = date
+    @book = book
+    book.rental_details << self
+    @person = person
+    person.rental_details << self
+  end
```

Comment:
- [ ] You are using `book` and `person` objects without passing them to the constructor. Kindly include `book` and `person` in the parameters.

  ```
  def initialize(date, book, person)
    ...
  end
  ```

Reply:
I have also implemented the change which was required here.